analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/NullPointerDereferenceTest.cs

```diff
@@ -18,47 +18,82 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using SonarAnalyzer.Common;
 using SonarAnalyzer.Rules.CSharp;
 
 namespace SonarAnalyzer.UnitTest.Rules.SymbolicExecution
 {
     [TestClass]
     public class NullPointerDereferenceTest
     {
-        private readonly VerifierBuilder sonar = new VerifierBuilder<SymbolicExecutionRunner>()
+        private readonly VerifierBuilder sonar = new VerifierBuilder()
+            .AddAnalyzer(() => new SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabledWithSonarCfg))
             .WithBasePath(@"SymbolicExecution\Sonar")
             .WithOnlyDiagnostics(new[] { NullPointerDereference.S2259 });
 
+        private readonly VerifierBuilder roslyn = new VerifierBuilder()
+            .AddAnalyzer(() => new SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabled))
+            .WithBasePath(@"SymbolicExecution\Roslyn")
+            .WithOnlyDiagnostics(new[] { NullPointerDereference.S2259 });
+
         [TestMethod]
-        public void NullPointerDereference_CS() =>
+        public void NullPointerDereference_Sonar_CS() =>
             sonar.AddPaths("NullPointerDereference.cs").WithConcurrentAnalysis(false).Verify();
 
         [TestMethod]
-        public void NullPointerDereference_DoesNotRaiseIssuesForTestProject() =>
+        public void NullPointerDereference_Roslyn_CS() =>
+            roslyn.AddPaths("NullPointerDereference.cs").WithConcurrentAnalysis(false).Verify();
```

Comment:
All copied test files (last PR) are analyzed now by the new Roslyn based checks.